web/index.php

```diff
@@ -108,43 +105,47 @@
             $queryBuilder
                 ->andWhere('class_name = :class')
                 ->setParameter(':class', 'Outlandish\Wpackagist\Package\Theme');
+            $wporgQueryResults = $wporgClient->getThemesBy('search', $query, $currentPage, 30, ['slug']);
+            $slugs = array_column($wporgQueryResults['themes'], 'slug');
             break;
         case 'plugin':
             $queryBuilder
                 ->andWhere('class_name = :class')
                 ->setParameter(':class', 'Outlandish\Wpackagist\Package\Plugin');
+            $wporgQueryResults = $wporgClient->getPluginsBy('search', $query, $currentPage, 30, ['slug']);
+            $slugs = array_column($wporgQueryResults['plugins'], 'slug');
             break;
         default:
             break;
     }
 
-    switch ($active) {
-        case 1:
-            $queryBuilder->andWhere('is_active');
-            break;
-
-        default:
-            $queryBuilder->orderBy('is_active', 'DESC');
-            break;
-    }
-
     if (!empty($query)) {
+        // We match the results from the wordpress API to our database.
         $queryBuilder
-            ->andWhere('name LIKE :name')
-            ->addOrderBy('name LIKE :order', 'DESC')
-            ->addOrderBy('name', 'ASC')
-            ->setParameter(':name', "%{$query}%")
-            ->setParameter(':order', "{$query}%");
```

Comment:
@lavoiesl it's actually the opposite, we get the slugs of the packages from WP API. We get them in the order they are shown on the actual website. We query those slugs from our database, and order them with PHP to match what we got from Wordpress (see the lines just below.)